notebook/analysis/benchmark_results.py

```diff
@@ -688,7 +693,266 @@ def return_unlabeled():
+tb_metrics
+
+# %%
+plot = ggplot(tb_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += geom_hline(yintercept=.5, linetype='dashed')
+plot += ggtitle('tb Crossover Point')
+plot
+
+# %% [markdown]
+# ## Lupus Analyses
+
+# %%
+in_files = glob.glob('../../results/keep_ratios.lupus*.tsv')
+in_files = [file for file in in_files if 'be_corrected' in file]
+print(in_files[:5])
+
+# %%
+lupus_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('lupus.')[-1]
+    model_info = model_info.split('.')
+        
+    supervised_model = model_info[0]
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-2]
+        
+    lupus_metrics = pd.concat([lupus_metrics, new_df])
+    
+lupus_metrics['train_count'] = lupus_metrics['train sample count']
+
+# Looking at the training curves, deep_net isn't actually training
+# I need to fix it going forward, but for now I can clean up the visualizations by removing it
+lupus_metrics = lupus_metrics[~(lupus_metrics['supervised'] == 'deep_net')]
+lupus_metrics['supervised'] = lupus_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
+lupus_metrics
+
+# %%
+plot = ggplot(lupus_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += geom_hline(yintercept=.5, linetype='dashed')
+plot += ggtitle('lupus Crossover Point')
+plot
+
+# %% [markdown]
+# ## Lupus Not Batch Effect Corrected
+
+# %%
+in_files = glob.glob('../../results/keep_ratios.lupus*.tsv')
+in_files = [file for file in in_files if 'be_corrected' not in file]
+print(in_files[:5])
+
+# %%
+lupus_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('lupus.')[-1]
+    model_info = model_info.split('.')
+        
+    supervised_model = model_info[0]
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-2]
+        
+    lupus_metrics = pd.concat([lupus_metrics, new_df])
+    
+lupus_metrics['train_count'] = lupus_metrics['train sample count']
+
+# Looking at the training curves, deep_net isn't actually training
+# I need to fix it going forward, but for now I can clean up the visualizations by removing it
+lupus_metrics = lupus_metrics[~(lupus_metrics['supervised'] == 'deep_net')]
+lupus_metrics['supervised'] = lupus_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
+lupus_metrics
+
+# %%
+plot = ggplot(lupus_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += geom_hline(yintercept=.5, linetype='dashed')
+plot += ggtitle('lupus Crossover Point')
+plot
```

Comment:
A bit of an unrelated question, but are you doing any sort of normalization/standardization on the input gene expression data? It looks like refine.bio does quantile normalization by default - you could try standardizing (taking z-scores of) each gene instead, although I'm not sure how much difference this would make vs. quantile normalization.

This might be more trouble than it's worth, though, if the data took a long time to download.

Reply:
I'm currently just using the quantile normalization from refine.bio. It may be worth trying other methods like z-scoring or [class-specific quantile normalization](https://www.nature.com/articles/s41598-020-72664-6.pdf).